src/VCAuthn/Controllers/WebhooksController.cs

```diff
@@ -0,0 +1,84 @@
+using System;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using VCAuthn.IdentityServer.SessionStorage;
+
+namespace VCAuthn.Controllers
+{
+    [Route("/topic")]
+    public class WebhooksController : ControllerBase
+    {
+        private readonly ISessionStorageService _sessionStorageService;
+        private readonly ILogger<WebhooksController> _logger;
+
+        public WebhooksController(ISessionStorageService sessionStorageService, ILogger<WebhooksController> logger)
+        {
+            _sessionStorageService = sessionStorageService;
+            _logger = logger;
+        }
+        
+        
+        [HttpPost("{topic}")]
+        public async Task<ActionResult> GetTopicUpdate(string topic)
+        {
+            if (String.Equals("presentations", topic, StringComparison.InvariantCultureIgnoreCase) == false)
+            {
+                _logger.LogDebug($"Skipping webhook for topic [{topic}]");
+                return Ok();
+            }
+            
+            string payload;
+            using (var reader = new StreamReader(Request.Body, Encoding.UTF8))
+            {  
+                payload = await reader.ReadToEndAsync();
+            }
+            _logger.LogInformation($"Topic [{topic}], Payload [{payload}]");
+
+            var update = JsonConvert.DeserializeObject<PresentationUpdate>(payload);
+
+            if (update.State != "presentation_received")
```

Comment:
I usually prefer to keep such values where they are used, rather than aggregating them in separate files. In that way, it's usually easier to read and debug.
However, I think you're right about this case because it will be consistent with the rest of the project.